src/components/App.tsx

```diff
@@ -0,0 +1,115 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import 'bulma/css/bulma.css';
+import '../App.scss';
+import Goodlist from './GoodList';
+import { SortField } from './SortField';
+
+export const goodsFromServer: string[] = [
+  'Dumplings',
+  'Carrot',
+  'Eggs',
+  'Ice cream',
+  'Apple',
+  'Bread',
+  'Fish',
+  'Honey',
+  'Jam',
+  'Garlic',
+];
+
+const sortAndReverseGoods = (
+  goods: string[],
+  sortField: SortField | '',
+  isReversed: boolean,
+): string[] => {
+  const sortedGoods = [...goods];
+
+  switch (sortField) {
+    case SortField.Alphabetically:
+      sortedGoods.sort((a, b) => a.localeCompare(b));
+      break;
+    case SortField.ByLength:
+      sortedGoods.sort((a, b) => a.length - b.length);
+      break;
+    default:
+  }
+
+  if (isReversed) {
+    sortedGoods.reverse();
+  }
+
+  return sortedGoods;
+};
+
+const App: React.FC = () => {
+  const [isReversed, setIsReversed] = useState<boolean>(false);
+  const [sortField, setSortField] = useState<SortField | ''>('');
```

Comment:
```suggestion
  const [sortField, setSortField] = useState<SortField>(SortField.Default);
```
better to add the empty field to this enum